portal-impl/src/content/Language_sv.properties

```diff
@@ -4981,8 +4981,8 @@ the-selected-role-cannot-be-deleted-because-it-is-a-required-system-role=Den val
 the-selected-web-content-no-longer-exists=Det valda innehållet existerar inte längre.
 the-sign-in-process-is-temporarily-disabled-because-the-portal-is-undergoing-a-routine-maintenance-upgrade=Inloggningsprocessen är tillfälligt inaktiverad eftersom portalen genomgår en underhållsuppgradering. Inloggningsprocessen återaktiveras när uppgraderingen är slutförd. Tack för ditt tålamod.
 the-signature-below-will-be-added-to-each-outgoing-message=Signaturen kommer läggas till på varje utgående meddelande.
-the-site-cannot-be-deleted-or-deactivated-because-it-is-a-required-system-site=Det går inte att ta bort eller inaktivera platsen eftersom det är en obligatorisk systemplats.
-the-site-cannot-be-deleted-or-deactivated-because-you-are-accessing-the-site=Det går inte att ta bort eller inaktivera webbplatsen eftersom du använder webbplatsen.
+the-site-cannot-be-deleted-or-deactivated-because-it-is-a-required-system-site=Det går inte att ta bort eller inaktivera platsen eftersom det är en obligatorisk systemplats.
+the-site-cannot-be-deleted-or-deactivated-because-you-are-accessing-the-site=Det går inte att ta bort eller inaktivera webbplatsen eftersom du använder webbplatsen.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.